src/theme.js

```diff
@@ -56,50 +53,31 @@ export const breakpoint = size => {
 }
 
 // ---------- COLOUR -----------
-const c_NEON_PINK = '#fb37f1'
-const c_NEON_PURPLE = '#9013fe'
-const c_NEON_BLUE = '#54fcfd'
-const c_NEON_YELLOW = '#fdf958'
-const c_NEON_GREEN = '#48e6b6'
-const c_BLACK = '#131313'
+const c_NEON_PINK = '#ff37ab'
+const c_NEON_PURPLE = '#5600ee'
+const c_BLACK = '#000'
 const c_WHITE = '#fff'
-const c_CLOUD_WHITE = '#effffa'
-const c_PURPLE = '#9013fe'
-const c_DARK_PURPLE = '#6d10c9'
 
-// Neons, shouldn't be used with some kind of randomisation, so only export an array
-export const c_NEONS = [
-  c_NEON_PINK,
-  c_NEON_PURPLE,
-  c_NEON_BLUE,
-  c_NEON_YELLOW,
-  c_NEON_GREEN,
-]
+// Text + background colours
+export const c_TEXT_LIGHT = c_WHITE
+export const c_TEXT_DARK = c_BLACK
 
-// Text + background colour combinations
-// PRIMARY
-export const c_PRIMARY_TEXT = c_WHITE
-export const c_PRIMARY_BACKGROUND = '#1f1e1f'
-export const c_PRIMARY_BACKGROUND_ALTERNATIVE = c_BLACK
-export const c_COOKIE_BACKGROUND = c_CLOUD_WHITE
-export const c_COOKIE_BUTTON = c_PURPLE
-export const c_COOKIE_BUTTON_HOVER = c_DARK_PURPLE
-// Secondary
-export const c_SECONDARY_TEXT = c_BLACK
+export const c_PRIMARY_BACKGROUND = c_BLACK
+export const c_COOKIE_BACKGROUND = c_WHITE
 export const c_SECONDARY_BACKGROUND = c_WHITE
 
 // UI
 export const c_CALL_TO_ACTION = c_NEON_PURPLE
-export const c_LOGO_BACKGROUND = c_WHITE
+export const c_CALL_TO_ACTION_HOVER = '#620aff'
```

Comment:
imo, if we used the value more than once, then it would make sense to have a layer of indirection.
I see we now also only use `c_NEON_PINK` and `c_NEON_PURPLE` once within the file, so I'd argue to remove the indirection for those too.

Reply:
Makes sense.